neqo-transport/src/recovery.rs

```diff
@@ -318,6 +318,14 @@ impl LossRecovery {
         self.cc.ssthresh()
     }
 
+    pub fn smoothed_rtt(&self) -> Option<Duration> {
+        self.rtt_vals.smoothed_rtt
+    }
+
+    pub fn set_smoothed_rtt(&mut self, value: Duration) {
+        self.rtt_vals.smoothed_rtt = Some(value)
+    }
+
```

Comment:
For this part, we need to not set the smoothed value, but instead overwrite the initial value (which is stored in latest_rtt).  That is currently a constant (INITIAL_RTT, right at the top of this file).

Reply:
Sorry, didn't get it first.
Fixed it. But I'm not sure that I did it right: I added set_latest_rtt for LossRecovery to overwrite the value in latest_rtt.